website/docs/modules/ROOT/pages/cli/installation-ide.adoc

```diff
@@ -65,6 +65,25 @@ folder with one of the Mill example projects. There are a wide range of example
 from hello-world to multi-module libraries to client-server web applications, and you can
 pick one most similar to what you are doing so you can hit the ground running working.
 
+=== Bash/Zsh Tab Completion
+
+You can set up Bash and Zsh tab completions by subsequently running the command:
+
+[source,console,subs="verbatim,attributes"]
+----
+> ./mill mill.tabcomplete/install
+----
+
+After that, doing `./mill <tab>` should begin showing possible modules and tasks for you
+to select at the command line.
+
+You can also choose to write the tab-completion script to a manually-specified path via:
+
+[source,console,subs="verbatim,attributes"]
+----
+> ./mill mill.tabcomplete/install --dest <path>
```

Comment:
We might need to say, that we don't modify shell config files if `--dest` is given.